README.md

```diff
@@ -47,9 +47,12 @@ $ qemu-system-i386 pinpog
 
 ### Windows
 
-[TBD](https://github.com/tsoding/pinpog/issues/65)
+1. Download UltraISO https://www.ultraiso.com/
+1. In nasm navigate to pinpong.asm and type in "nasm pinpog.asm -o pinpog.bin".
+1. Open UltraISO and open the .bin file by going to New->Bootable CD/DVD Image.
```

Comment:
Why do we have to create ISO in the first place? Can we just use pinpog.bin directly like on Linux?